ros2/cc_defs.bzl

```diff
@@ -109,12 +109,13 @@ def ros2_cpp_binary(name, ros2_package_name = None, set_up_ament = False, **kwar
         name: A unique target name.
         ros2_package_name: If given, defines a ROS package name for the target.
             Otherwise, the `name` is used as the package name.
-        set_up_ament: If true, sets up ament file tree for the binary target.
+        set_up_ament: If true, wrap the binary target to set up ament file tree and
+            missing ROS_HOME/ROS_LOG_DIR env vars.
```

Comment:
Setting up ROS_HOME+ROS_LOG_DIR is only needed for tests, IMO. This comment can be reverted. In addition, the logic in launch.sh.tpl only sets up those env vars for tests.

Reply:
true